_docs_integrate/data-model-overview.md

```diff
@@ -94,6 +94,7 @@ A Relationship between two Identities is the prerequisite for them to exchange M
 | peer             | `string`                                                                                                                                         | The `address` of the [Identity](#identity) with which you have this Relationship.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | saved only locally                            |
+| peerIdentity     | [`Identity`](#identity)                                                                                                                          | The [Identity](#identity) with which you have this Relationship.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | saved only locally                            |
 | peerDeletionInfo | [`PeerDeletionInfo`](#peerdeletioninfo) \| `undefined`                                                                                           | Information about whether the Identity with which you have this Relationship is to be deleted or was deleted, as well as the point in time of that deletion.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  | saved only locally                            |
```

Comment:
For `peer` as well as `peerIdentity` and `peerDeletionInfo` of the Relationship, the remark "saved only locally" is now stated. However, the Backbone must know between which Identities a Relationship exists and also know the state of deletion of an Identity to a minimum extent.

@jkoenig134 Can you perhaps help me to find out whether the remarks are still okay like this or how they can be corrected or made more precise?

Reply:
ehm, I am confused .. I think that backbone knows all of this information?

Reply:
@tnotheis this column was your idea, what was the plan about this? Should this represent that this data is not stored **exactly** like this in the backbone? B/c we expand this and save it in the db.

Reply:
Yeah, it was already debatable back then when I introduced it. The idea was to let the reader know what information the backbone has. But tbh: this list can never be complete anyway. So we should probably delete the column.